config/thisroot.sh

```diff
@@ -162,6 +166,14 @@ set_environment()
    else
       JUPYTER_CONFIG_DIR=$ROOTSYS/etc/notebook:$JUPYTER_CONFIG_DIR; export JUPYTER_CONFIG_DIR
    fi
+
+   if [ -z "${ROOT_INCLUDE_PATH-}" ]; then
+      ROOT_INCLUDE_PATH=@DEFAULT_ROOT_INCLUDE_PATH@
+      export ROOT_INCLUDE_PATH
+   else
+      ROOT_INCLUDE_PATH=@DEFAULT_ROOT_INCLUDE_PATH@:$ROOT_INCLUDE_PATH
```

Comment:
Do you consider this an issue if DEFAULT_ROOT_INCLUDE_PATH is empty? Not just here but in the other shell setups?

Reply:
I don't think this should be an issue. This case is also covered by all GHA runners.